packages/typespec-ts/src/utils/clientUtils.ts

```diff
@@ -85,7 +85,7 @@ export function getModularClientOptions(
 ) {
   const [hierarchy, client] = clientMap;
   const clientOptions: ModularClientOptions = {
-    rlcClientName: `${client.name.replace("Client", "")}Context`
+    rlcClientName: `${client.name.replace(/Client$/, "")}Context`
```

Comment:
classic client import file not exist. eg:  `import locationClientContext.js` but generation file is `clientLocationContext.ts`